modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomViews.path

```diff
@@ -292,7 +292,7 @@
 	</tr>
 	<tr>
 		<td>FIELD_VIEW_FILTER_DATE_INPUT</td>
-		<td>//div[@class='col-lg-6'][${key_dateIndex}]//div[contains(@class, 'input-group-item')]/input[contains(@placeholder,'YYYY-MM-DD')]</td>
+		<td>//div[@class='col-lg-6'][${key_dateIndex}]//div[contains(@class, 'input-group-item')]/input[contains(@placeholder,'__/__/____')]</td>
```

Comment:
The [DateTime](https://issues.liferay.com/browse/LPS-143068) story must have affected something in the Aggregation Filter. Previously the DD-MM-YYYY format was used. But now the month/day/year format is being used here.